cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -574,9 +574,10 @@ func kubeClientGet(uc *UpstreamController, namespace string, name string, queryT
 		klog.Error(err)
 		return nil, err
 	}
-	if err == nil && obj != nil {
-		util.SetMetaType(obj.(runtime.Object))
+	if err != nil {
+		return obj, err
 	}
+	err = util.SetMetaType(obj.(runtime.Object))
```

Comment:
`SetMetaType` should return obj as nil on error.
If you want to ignore errors, don't assign err.

Reply:
Or obj is not nil, is there any special scenario?

Reply:
> Or obj is not nil, is there any special scenario?

This pr try to fix go lint error, which reports "unhandle error", so we should assign this err explicitly.

Reply:
> `SetMetaType`错误时将 obj 返回为 nil。 如果您想分配错误，请不要分配错误。

Will this bring any new problems?

Reply:
will not

Reply:
```suggestion
	if err := util.SetMetaType(obj.(runtime.Object)); err != nil {
	    return nil, err
	}
```

Reply:
updated